pxr/usdImaging/usdImagingGL/testenv/testUsdImagingGLMaterialX/basicMxYup.usda

```diff
@@ -22,7 +22,7 @@ def Xform "lights"
 {
     def DomeLight "DomeLight"
     {
-        asset inputs:texture:file = "./Env_StinsonBeach.tex"
+        asset inputs:texture:file = @./StinsonBeach.hdr@
```

Comment:
I was not able to get the `Env_StinsonBeach.tex` to load which caused most of the MaterialX tests to fail. I copied `StinsonBeach.hdr` from [hdx](https://github.com/PixarAnimationStudios/USD/blob/dev/pxr/imaging/hdx/textures/StinsonBeach.hdr) and updated the asset path. Doing so allowed the MaterialX tests to pass. Is there a specific plugin needed for `.tex` files?

Reply:
I think that's a good change, @nvidia-jomiller - only RenderMan and an pixar-internal Hio image plugin can read .tex files.  That plugin is always active in our codebase, which is why we didn't catch this - .tex definitely does not belong in usdImagingGL !